### ISSUE TITLE
"Deployment Failed with Error: Malicious entry" using AzureFunctionAppV1 1.0.4

### ISSUE THREAD

**Issue by NatMarchand** [Microsoft/azure-pipelines-tasks#10010]
## Required Information
**Question, Bug, or Feature?**  
*Type*: Bug

**Enter Task Name**: AzureFunctionAppV1 1.0.4

## Environment
Azure pipeline release : https://dev.azure.com/natmarchand/github/_releaseProgress?releaseId=24&environmentId=24&_a=release-environment-logs
Hosted agent queue : Hosted Windows 2019 with VS2019

## Issue Description
I'm deploying an Azure Function v2 built with C# and Run From Package technology.
The task fails with error :
```
2019-04-02T17:37:01.0704697Z ##[debug]Deployment Failed with Error: Error: Malicious entry: Answer\function.json
2019-04-02T17:37:01.0704983Z ##[debug]task result: Failed
2019-04-02T17:37:01.0763607Z ##[error]Error: Malicious entry: Answer\function.json
2019-04-02T17:37:01.0771812Z ##[debug]Processed: ##vso[task.issue type=error;]Error: Malicious entry: Answer\function.json
2019-04-02T17:37:01.0772474Z ##[debug]Processed: ##vso[task.complete result=Failed;]Error: Malicious entry: Answer\function.json
```
The first attempt using the task version 1.0.2 of this release was successful. Replaying the release use the version 1.0.4 and fails with the error.

**Comment by vincentdass** [comment]
@NatMarchand , thanks for reporting the issue. We have identified the issue and working on the fix.

**Comment by NatMarchand** [comment]
Any ETA of the fix in production ? I still have the task v1.0.4 and stuck not able to deploy with my pipeline.

**Comment by vincentdass** [comment]
@NatMarchand , Apologies, the deployment is in progress and the fix should be available by this week.
